grpc/sa-wrappers.go

```diff
@@ -283,6 +283,26 @@ func (sac StorageAuthorityClientWrapper) PreviousCertificateExists(
 	return exists, err
 }
 
+func (sac StorageAuthorityClientWrapper) AddPrecertificate(
+	ctx context.Context,
+	req *sapb.AddCertificateRequest,
+) (*corepb.Empty, error) {
+	if req == nil {
+		return nil, errIncompleteResponse
```

Comment:
I think this should be `errIncompleteRequest` no?
```suggestion
		return nil, errIncompleteRequest
```